### ISSUE TITLE
django-environ added as Python dependency

### ISSUE THREAD

**Issue by frafra** [NINAnor/python-template#17]
https://github.com/NINAnor/python-template/blob/6c8cebe5d4b316c7b74c6192d5454b2de166c77c/python/pyproject.toml.jinja#L8

Do we really want it even for non Django project?

**Comment by nicokant** [comment]
Yes, despite the name of the package it's not django dependent or strictly django related. It's also used in the `main.py` to read the environment variables

**Comment by frafra** [comment]
What about https://hydra.cc/?

**Comment by frafra** [comment]
Hydra is great for a complex script. We can keep django-environ as base dependency.
